x/config/config_test.go

```diff
@@ -14,13 +28,12 @@ func TestSanitizeConfig(t *testing.T) {
 
 	// Test that a invalid cypher is rejected.
 	sanitizedConfig, err := SanitizeConfig("split:5|ss://jhvdsjkfhvkhsadvf@example.com:1234?prefix=HTTP%2F1.1%20")
-	require.NoError(t, err)
-	require.Equal(t, "split:5|ss://ERROR", sanitizedConfig)
+	require.Error(t, err)
```

Comment:
What would be the `sanitizedConfig` in this case? Is it `nil`?

Reply:
If we can't sanitize, the call fails, instead of returning ERROR. I think that's less surprising.